android/app/src/main/java/org/fossasia/openevent/status/BookmarkStatus.java

```diff
@@ -0,0 +1,18 @@
+package org.fossasia.openevent.status;
+
+import android.widget.ImageButton;
+
+import org.fossasia.openevent.listeners.OnBookmarkSelectedListener;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class BookmarkStatus {
+
+    private ImageButton imageButton;
```

Comment:
Why can't we handle the image button locally, why do we need to send it in a listener? We should only propagate data, not views.

Reply:
`slotBookmark` belongs to the viewholder, so I cannot access it outside without passing it. Or I will have to access the view holder somehow.

Reply:
Why do you need to access it in the listener implementation?

Reply:
I need to update the UI in the snackbar action for undo.

Reply:
When you undo, the view is updated again, right? So it can be updated there